docs/self-service/README.md

```diff
@@ -223,3 +223,58 @@ DEV_IMG ?= # <== NOTE:  outside dev, change this!!!
 ```
 
 During the build targets, the necessary config file gets updated with this image name.  Then once the yaml targets are use, provider-components.yaml will contain the desired controller container image.
+
+## Troubleshooting Resources to Aid in Resolving Provisioning Failures
+
+1. Ensure `provider-components.yaml` specifies a container image for the `vsphere-provider-controller-manager` Statefulset which has ClusterAPI Provider vSphere v0.2.0. or later.  If one is not available then one can be created using the `make dev-build` process.
+2. After running `clusterctl create cluster...`, verify the two ClusterAPI Provider vSphere pods `vsphere-provider-controller-manager-0` and `cluster-api-controller-manager-0` are running in the bootstrap cluster:
+    ```shell
+    $> kubectl get pods --all-namespaces
+    NAMESPACE                 NAME                                         READY   STATUS    RESTARTS   AGE
+    cluster-api-system        cluster-api-controller-manager-0             1/1     Running   0          5d1h
+    kube-system               coredns-fb8b8dccf-bs2v9                      1/1     Running   0          5d1h
```

Comment:
i suggest to remove lists those kube-system pods, just focus on the two relevant cluster-api pods, how do you think?